CHANGELOG.md

```diff
@@ -6,9 +6,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-## [0.8.0] - 2021-02-09
-### Added
-- `escapeLinkRegex` useful option to use a regexp to remove unwanted characters.
```

Comment:
The `### Added` section and the `description` of what you've done should stay in the PR.

Reply:
```md
### Added
- `escapeLinkRegex` useful option to use a regexp to remove unwanted characters.
```